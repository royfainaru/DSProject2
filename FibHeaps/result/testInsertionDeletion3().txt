╮
├── ╭────────────────╮
│   │ Key: 3         │
│   ├────────────────┤
│   │ Rank: 2        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 8        │
│   │ Prev: (null)   │
│   │ Child: 15      │
│   ╰────────────────╯
│   ├── ╭───────────────╮
│   │   │ Key: 15       │
│   │   ├───────────────┤
│   │   │ Rank: 1       │
│   │   │ Marked: false │
│   │   │ Parent: 3     │
│   │   │ Next: 100     │
│   │   │ Prev: (null)  │
│   │   │ Child: 18     │
│   │   ╰───────────────╯
│   │   ╰── ╭───────────────╮
│   │       │ Key: 18       │
│   │       ├───────────────┤
│   │       │ Rank: 0       │
│   │       │ Marked: false │
│   │       │ Parent: 15    │
│   │       │ Next: (null)  │
│   │       │ Prev: (null)  │
│   │       │ Child: (null) │
│   │       ╰───────────────╯
│   ╰── ╭───────────────╮
│       │ Key: 100      │
│       ├───────────────┤
│       │ Rank: 0       │
│       │ Marked: false │
│       │ Parent: 3     │
│       │ Next: (null)  │
│       │ Prev: 15      │
│       │ Child: (null) │
│       ╰───────────────╯
╰── ╭────────────────╮
    │ Key: 8         │
    ├────────────────┤
    │ Rank: 1        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: (null)   │
    │ Prev: 3        │
    │ Child: 20      │
    ╰────────────────╯
    ╰── ╭───────────────╮
        │ Key: 20       │
        ├───────────────┤
        │ Rank: 0       │
        │ Marked: false │
        │ Parent: 8     │
        │ Next: (null)  │
        │ Prev: (null)  │
        │ Child: (null) │
        ╰───────────────╯
