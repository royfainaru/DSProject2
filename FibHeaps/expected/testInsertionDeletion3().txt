╮
├── ╭────────────────╮
│   │ Key: 8         │
│   ├────────────────┤
│   │ Rank: 1        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 3        │
│   │ Prev: 3        │
│   │ Child: 20      │
│   ╰────────────────╯
│   ╰── ╭───────────────╮
│       │ Key: 20       │
│       ├───────────────┤
│       │ Rank: 0       │
│       │ Marked: false │
│       │ Parent: 8     │
│       │ Next: 20      │
│       │ Prev: 20      │
│       │ Child: (null) │
│       ╰───────────────╯
╰── ╭────────────────╮
    │ Key: 3         │
    ├────────────────┤
    │ Rank: 2        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: 8        │
    │ Prev: 8        │
    │ Child: 15      │
    ╰────────────────╯
    ├── ╭───────────────╮
    │   │ Key: 15       │
    │   ├───────────────┤
    │   │ Rank: 1       │
    │   │ Marked: false │
    │   │ Parent: 3     │
    │   │ Next: 100     │
    │   │ Prev: 100     │
    │   │ Child: 18     │
    │   ╰───────────────╯
    │   ╰── ╭───────────────╮
    │       │ Key: 18       │
    │       ├───────────────┤
    │       │ Rank: 0       │
    │       │ Marked: false │
    │       │ Parent: 15    │
    │       │ Next: 18      │
    │       │ Prev: 18      │
    │       │ Child: (null) │
    │       ╰───────────────╯
    ╰── ╭───────────────╮
        │ Key: 100      │
        ├───────────────┤
        │ Rank: 0       │
        │ Marked: false │
        │ Parent: 3     │
        │ Next: 15      │
        │ Prev: 15      │
        │ Child: (null) │
        ╰───────────────╯
